debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/operatorutil/OperatorUtil.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.testing.system.tools.operatorutil;
+
+import static io.debezium.testing.system.tools.ConfigProperties.PRODUCT_BUILD;
+
+import io.debezium.testing.system.tools.OpenShiftUtils;
+import io.debezium.testing.system.tools.kafka.builders.StrimziSubscriptionBuilder;
+import io.debezium.testing.system.tools.registry.builders.ApicurioSubscriptionBuilder;
+import io.fabric8.openshift.api.model.operatorhub.v1alpha1.Subscription;
+import io.fabric8.openshift.client.OpenShiftClient;
+
+/**
+ * Methods shared in Strimzi and Apicurio operator deployment
+ */
+public class OperatorUtil {
+    private final static String STABLE = "stable";
+
+    /**
+     * Construct operator subscription instance and install operator to given namespace and wait for its install
+     * @param ocp
+     * @param operatorEnum
+     * @param namespace
+     * @throws InterruptedException
+     */
+    public static void deployOperator(OpenShiftClient ocp, OpenshiftOperatorEnum operatorEnum, String namespace) throws InterruptedException {
+        OpenShiftUtils utils = new OpenShiftUtils(ocp);
+        utils.createOrReplaceOperatorGroup(namespace, operatorEnum.getOperatorGroupName());
+
+        String channel = operatorEnum.getSubscriptionUpdateChannel();
+
+        Subscription subscription;
+        if (operatorEnum == OpenshiftOperatorEnum.STRIMZI && PRODUCT_BUILD) {
+            subscription = StrimziSubscriptionBuilder.base()
+                    .withProductConfig()
+                    .withNamespace(namespace)
+                    .withChannel(channel)
+                    .build();
+        }
+        else if (operatorEnum == OpenshiftOperatorEnum.STRIMZI) {
+            subscription = StrimziSubscriptionBuilder.base()
+                    .withCommunityConfig()
+                    .withNamespace(namespace)
+                    .withChannel(channel)
+                    .build();
+        }
+        else if (operatorEnum == OpenshiftOperatorEnum.APICURIO && PRODUCT_BUILD) {
+            subscription = ApicurioSubscriptionBuilder.base()
+                    .withProductConfig()
+                    .withNamespace(namespace)
+                    .withChannel(channel)
+                    .build();
+        }
+        else {
+            subscription = ApicurioSubscriptionBuilder.base()
+                    .withCommunityConfig()
+                    .withNamespace(namespace)
+                    .withChannel(channel)
+                    .build();
+        }
```

Comment:
I would refactor this part of code with 4 if-else branches and instead use some nested ifs with duplicate code extracted

Reply:
Good catch with the zip files I did not catch that.